.circleci/config.yml

```diff
@@ -21,12 +21,12 @@ jobs:
           name: Install server modules
           command: npm install --production
       - save_cache:
-          key: server_modules_{{ checksum "server/package.json" }}
+          key: server_modules_node_6_{{ checksum "server/package.json" }}
```

Comment:
The node version was because people do stupid "pull dependent on node_version" algorithms in their packages.